### ISSUE TITLE
External links don't open from global E or F buttons

### ISSUE THREAD

**Issue by nearnshaw** [decentraland/unity-renderer#109]
Opening external links is a restricted action that intentionally can only be triggered from a user's action to avoid spamming.

We allowed it to be called from OnPointerDown events and it seems also from click events
BUT, it's reported that it can't be called from a global E or F event.

This seems arbitrarily inconsistent. It's for example a good use case to be able to call an external link from an NPC interaction, and there it would work with E and F global events too.

As reported, this works:

```
Input.instance.subscribe('BUTTON_DOWN', ActionButton.POINTER, true, (e) => {
  openExternalURL("www.google.com")
}
```

This doesn't:

```
Input.instance.subscribe('BUTTON_DOWN', ActionButton.PRIMARY, true, (e) => {
  openExternalURL("www.google.com")
}
```

**Comment by kuruk-mm** [comment]
I tested and it can't be called from any of the globals events

Code tested:
```
Input.instance.subscribe('BUTTON_DOWN', ActionButton.POINTER, true, evt => {
  log("POINTER")
  openExternalURL("www.google.com")
})

Input.instance.subscribe('BUTTON_DOWN', ActionButton.PRIMARY, true, evt => {
  log("PRIMARY")
  openExternalURL("www.yahoo.com")
})

Input.instance.subscribe('BUTTON_DOWN', ActionButton.SECONDARY, true, evt => {
  log("SECONDARY")
  openExternalURL("www.youtube.com")
})

Input.instance.subscribe('BUTTON_DOWN', ActionButton.ANY, true, evt => {
  log("ANY")
  openExternalURL("www.twitter.com")
})
```

And I get:
```
Error: openExternalUrl can only be used inside a pointerEvent
```

In all the cases.

That is the desired behavior?

**Comment by nearnshaw** [comment]
@kuruk-mm
I just tested this with a fresh scene and you're right, none of those work
I think the pointer global event did work back when I reported this, but it was a while ago so things can change 🤷 

I think it's ok that this stays as it is, then. The problem was the arbitrary inconsistency, but if we're consistent with how we treat all buttons, that's fine. And yes, there's definitively chance for abuse there if clicking anywhere opens a link. We want to guard agains those scenarios, but there's always a tradeoff between that and freedom for creators. In this case, I don't have strong opinions over which side to favor.

In any case, out of all the HUGE list of issues we have with the SDK, I would probably put this one somewhere in the bottom 10 of the list, it really shouldn't be a priority at all. I say we close it.
